go/libraries/doltcore/diff/diff.go

```diff
@@ -40,6 +41,14 @@ const (
 	ModifiedNew
 )
 
+var ColDiffColors = map[ChangeType]*color.Color{
```

Comment:
You didn't create this problem, but this is the wrong layer in the stack to encode color information for display. Should live in the CLI package, and get passed into diff writers at construction time so they know what colors to write.